lib/helpers/response.ex

```diff
@@ -0,0 +1,30 @@
+defmodule RogerUi.Helpers.Response do
+  @moduledoc """
+  CORS and JSON
+  """
+  import Plug.Conn
+
+  defp set_cors_headers(conn) do
```

Comment:
Can you place private functions at the bottom? I think it makes sense to first display the public API first and then the implementation.

Reply:
All private functions moved to end of module